source/deployment/ldap-group-constrained-team-channel.rst

```diff
@@ -0,0 +1,61 @@
+3. Ensure **Team Settings > General > Allow any user with an account on this server to join this team** is set to ``No``. 
+4. Convert the team to have its membership managed by synchronized groups by running the `group team enable CLI command <https://docs.mattermost.com/administration/command-line-tools.html#mattermost-group-team-enable>`_ .
+
+To convert a private channel: 
+
+1. Ensure there is at least one group already associated to the channel. You can view and add default channels to a group via **System Console > User Management > Groups > Group Configuration**. Please see more information on adding default teams and channels `here <https://docs.mattermost.com/deployment/ldap-group-sync.html#add-default-teams-or-channels-for-the-group>`_. Additionally, you can use the CLI tool to view if there is already a group associated to the channel by running the `group channel list CLI command <https://docs.mattermost.com/administration/command-line-tools.html#mattermost-group-team-list>`_. 
+2. Convert the team to have its membership managed by synchronized groups by running the `group channel enable CLI command <https://docs.mattermost.com/administration/command-line-tools.html#mattermost-group-channel-enable>`_ .  
+
+.. note:: 
+   System console support to set teams and private channels to managing membership with group synchronization is coming estimated in the 5.14 (August 2019) release. 
+
+Add or Remove Groups from Teams
+----------
+
+Once the management of the team is converted to be managed by synchronized groups, a team admin can add additional groups from **Main Menu > Add Groups to Team**.  This will add users on the next AD/LDAP synchronization and any new users to the group will be added to the team on subsequent synchronizations. 
+
+Team administrators can also remove groups from a team from **Main Menu > Manage Groups.  This will disassociate the group from the team.  Users are removed on the next AD/LDAP synchronization. 
+
+Add or Remove Groups from Private Channels
+----------
+
+Once the management of the channel is converted to be managed by synchronized groups, a team admin can add additional groups from **Channel Menu > Add Groups to Channel**.  This will add users on the next AD/LDAP synchronization and any new users to the group will be added to the channel on subsequent synchronizations. 
+
+Team administrators can also remove groups from a team from **Main Menu > Manage Groups.  This will disassociate the group from the team.  Users are removed on the next AD/LDAP synchronization. 
```

Comment:
```suggestion
Team administrators can also remove groups from a team from **Main Menu > Manage Groups**. This will disassociate the group from the team. Users are removed on the next AD/LDAP synchronization. 
```